src/components/TokenSafety/index.tsx

```diff
@@ -255,10 +255,9 @@ export default function TokenSafety({
           <ShortColumn>
             <SafetyLabel warning={displayWarning} />
           </ShortColumn>
-          <ShortColumn>{heading && <InfoText fontSize="20px">{heading}</InfoText>}</ShortColumn>
           <ShortColumn>
             <InfoText>
-              {description}{' '}
+              {heading} {description}{' '}
```

Comment:
could you make the spaces more explicit with a template literal? just feels a bit brittle like this